extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -903,8 +903,12 @@ export class AmpStoryPage extends AMP.BaseElement {
    * @private
    */
   isAutoplaySupported_() {
-    VideoUtils.resetIsAutoplaySupported();
-    return VideoUtils.isAutoplaySupported(this.win, getMode(this.win).lite);
+    // We should not be detecting support each time, since it's slow.
+    // However, https://go.amp.dev/pr/23754 started resetting cached support
+    // checks likely to work around a race condition.
+    // TODO(alanorozco): Fix issue with cached `isAutoplaySupported`, and use
+    // that instead of `detectIsAutoplaySupported`.
+    return detectIsAutoplaySupported(this.win);
```

Comment:
@gmajoulet @Enriqe 

Resetting cache and checking again has the same effect as newly detecting each time, so I'm doing that instead.

However, I'm curious about the issue that prompted introducing `resetIsAutoplaySupported` on #23754. There may be an underlying race condition in whatever cached result.

Reply:
Checked a story document with autoplay video and we get the same result on Power Saving mode whether we reset or not.

Reply:
LGTM, thank you!